### ISSUE TITLE
how with Wholesale price?

### ISSUE THREAD

**Issue by ToanVenota** [kilbot/WooCommerce-POS#184]
Hello,

I would like to create the option for using wholesale price or not.

For example the regular price is 100, when the customer with wholesale price come, I just tick the checkbox "Wholesale" in POS screen, than all product will have a lower price by certain percent.

Do anybody know how to make it works?

Best regards,

**Comment by graffxdesign** [comment]
Ever figure this out? I am looking to do that or just force a customer role price. I would LOVE some help.

**Comment by gocasals** [comment]
@graffxdesign @ToanVenota  could you resolve this problem? I need th same solution.

**Comment by graffxdesign** [comment]
@gocasals no, I just built a second store with only wholesale pricing, stupid, why can't anyone build a good pos?
